bin/ch-docker2tar

```diff
@@ -16,7 +16,10 @@ EOF
 )
 
 parse_basic_args "$@"
-
+if [ "$1" = --nocompress ]; then
+    nocompress=yes
+    shift
+fi
```

Comment:
What if the argument is passed as such: `ch-dorcker2tar candle /var/tmp/foo --nocompress`?

Reply:
The --nocompress flag is positional. As it is likely only going to be used by the squashfs workflow or developers I don't expect users to need to know it is positional. I'm open to solutions to make the argument non positional if you would like to change it.